### DISCUSSION TITLE
Welcome to chunks-rs Discussions!

### DISCUSSION THREAD

**Discussion by sashk4** [sashk4/chunks-rs#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VimYoung** [comment]
Hello. I am Bhaumik, thanks for the great project @drkrssll :clap: :clap:  , it really got great potential :rocket: . I was about to mention you to display your work on reddit and other mediums for others to know. I am glad you mentioned me for my tiny contribution, I didn't do good enough till yet for you to place me in readme :sweat_smile: .
Also the PRs I pulled were my first,so thanks for it too, you helped open up a new FOSS contributor. As for some future work, I will try to pull more requests for adding more widgets. Also, I am thinking of adding some docs too, let's see home much I can squeeze of my leisure time. :raised_hands: :raised_hands:
